.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     name: Build the thrift compiler
     strategy:
       matrix:
-        os: [ubuntu-20.04, macos-12]
+        os: [ubuntu-20.04, macos-12, macos-14]
```

Comment:
is `macos-14` always a arm machine? not a intel one 🤔

Reply:
Yes => https://github.blog/changelog/2024-01-30-github-actions-macos-14-sonoma-is-now-available/ and it can also be confirmed on the runner details

`Current runner version: '2.316.1'
Operating System
Runner Image
  Image: macos-14-arm[6](https://github.com/upfluence/thrift/actions/runs/9370263535/job/25796793876?pr=11#step:1:7)4`